### ISSUE TITLE
Restrict-MDEDomain error

### ISSUE THREAD

**Issue by tora1104** [Azure/Azure-Sentinel#7179]
Running into this error when this playbook runs. 

![image](https://user-images.githubusercontent.com/51537003/215552544-efc11abc-1ba4-4aa8-8ec4-cd1b37cfdc8d.png)

Here's the error:

{
    "errors": [
        {
            "message": "Required properties are missing from object: $id, Type.",
            "lineNumber": 0,
            "linePosition": 0,
            "path": "[0]",
            "value": [
                "$id",
                "Type"
            ],
            "schemaId": "#/items",
            "errorType": "required",
            "childErrors": []
        },
        {
            "message": "Required properties are missing from object: $id, Type.",
            "lineNumber": 0,
            "linePosition": 0,
            "path": "[1]",
            "value": [
                "$id",
                "Type"
            ],
            "schemaId": "#/items",
            "errorType": "required",
            "childErrors": []
        }
    ]
}

Any help would be much appreciated!

**Comment by v-vdixit** [comment]
Hi @tora1104 thanks for flagging this, we are working with the team on this, will update you shortly.

**Comment by v-vdixit** [comment]
Hi @tora1104 we are working on this with the concerned tea, will update you shortly, thanks!

**Comment by v-vdixit** [comment]
Hi @tora1104 we already have a PR with the related updates, will be published once review is complete, thanks!

**Comment by tora1104** [comment]
Thank you for the updates!

**Comment by v-vdixit** [comment]
Hi @tora1104 the PR is still under review by team, will be published once merged, thanks!

**Comment by v-vdixit** [comment]
Hi @tora1104 the PR is being updated for other required changes, will be merged once review is complete, thanks!

**Comment by v-vdixit** [comment]
Hi @tora1104 we are working on changes requested by reviewer on PR, will be merged once review is complete, thanks!

**Comment by v-rbajaj** [comment]
Hi @tora1104, we are raising a new PR for this issue, we have closed the older because there are few more developments pending and which will be added.

**Comment by v-vdixit** [comment]
Hi @tora1104 we will be raising the PR with all updated changes and a few new additions till Friday, 21st July 2023, thanks!

**Comment by tora1104** [comment]
Is there a new file I should download? If so, where may I find it? Thanks!

**Comment by tora1104** [comment]
Nevermind, it's working today! I'll chalk it up to gremlins!

Thank you for getting this working again! Much appreciated!
